tests/func/test_get.py

```diff
@@ -36,6 +36,31 @@ def test_get_repo_dir(repo_dir, erepo):
     trees_equal(src, dst)
 
 
+def test_get_regular_file(erepo):
+    src = os.path.join(erepo.root_dir, "file")
```

Comment:
This must be relative path, that is also the reason for strange behaviour mentioned by @Suor  [here](https://github.com/iterative/dvc/pull/2837/files#r350940085).
Also probably an idea for new issue ticket: forbid the user from providing the full path in `get`.

Reply:
Also, erepo already has few things inside that can be `get`-ed:
for example:
in case of file: `Repo.get(erepo.root_dir, erepo.FOO, "foo_imported")`
in case of dir: `Repo.get(erepo.root_dir, erepo.DATA_DIR, "dir_imported")`

Reply:
> Also probably an idea for new issue ticket: forbid the user from providing the full path in get.

There was a ticket recently from a user that uses full path to get his external local output like that, so we can't really forbid it for good 🙂

Reply:
@pared 
>Also, erepo already has few things inside that can be get-ed:
for example:
in case of file: Repo.get(erepo.root_dir, erepo.FOO, "foo_imported")
in case of dir: Repo.get(erepo.root_dir, erepo.DATA_DIR, "dir_imported")

Yes, but those are `dvc add`ed in `erepo` fixture and this PR is aiming to test `get`ing files that are tracked by git and not dvc.

Reply:
Ahh right, sorry. Still, for a file that could be `erepo.code` for example.

Reply:
@pared sure, but we also need to test a dir too :)